gax/src/streamingCalls/streaming.ts

```diff
@@ -136,173 +148,71 @@ export class StreamProxy extends duplexify implements GRPCCallResult {
     }
   }
 
-  retry(stream: CancellableStream, retry: RetryOptions) {
-    let retryArgument = this.argument! as unknown as RequestType;
-    if (typeof retry.getResumptionRequestFn! === 'function') {
-      const resumptionRetryArgument =
-        retry.getResumptionRequestFn(retryArgument);
-      if (resumptionRetryArgument !== undefined) {
-        retryArgument = resumptionRetryArgument;
-      }
-    }
-    this.resetStreams(stream);
-
-    const newStream = this.apiCall!(
-      retryArgument,
-      this._callback
-    ) as CancellableStream;
-    this.stream = newStream;
-
-    this.streamHandoffHelper(newStream, retry);
-    return newStream;
-  }
-
   /**
    * Helper function to handle total timeout + max retry check for server streaming retries
    * @param {number} deadline - the current retry deadline
    * @param {number} maxRetries - maximum total number of retries
    * @param {number} totalTimeoutMillis - total timeout in milliseconds
+   * @param {GoogleError} originalError - underlying error received by the stream
    */
```

Comment:
Nit: Could we add comments for `orignialTimeout` and `retries` as well?

Reply:
great catch, totally did this docstring before i added those in